.github/workflows/gradle-wrapper-validation.yml

```diff
@@ -18,4 +18,4 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - uses: gradle/wrapper-validation-action@v3
+      - uses: gradle/actions/wrapper-validation@v4
```

Comment:
An small bonus to update this workflow to use the new wrapper-validation action.